### ISSUE TITLE
.evans.toml cannot specify port

### ISSUE THREAD

**Issue by terashi58** [ktr0731/evans#133]
When I put a `.evans.toml` file in the current directory like below,

```.toml
[server]
host = "192.168.0.5"
port = "8888"
```

and run `evans foo.proto`, then the port setting is ignored.

```
$ evans foo.proto
default@192.168.0.5:50051>
```

I believe the flag default value overrides the config file.

**Comment by ktr0731** [comment]
Thank you for reporting a bug! I'm going to fix it in the nearly future.

**Comment by ktr0731** [comment]
This bug is fixed at #145. Please update Evans to v0.6.11.
